internal/da/da_test.go

```diff
@@ -129,6 +129,62 @@ func TestDaBuildAndCommonPrefixSearch03(t *testing.T) {
 	}
 }
 
+func TestDaBuildAndCommonPrefixSearchCallback01(t *testing.T) {
+	d, err := Build(nil)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	d.CommonPrefixSearchCallback("", func(id, l int) {
+		t.Errorf("unexpected callback, id:%v, l:%v")
+	})
+}
+
+func TestDaBuildAndCommonPrefixSearchCallback02(t *testing.T) {
+	d, err := Build([]string{})
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	d.CommonPrefixSearchCallback("", func(id, l int) {
+		t.Errorf("unexpected callback, id:%v, l:%v")
```

Comment:
__[gometalinter]__
* [vet] missing argument for Errorf("%v"): format reads arg 1, have only 0 args